### PR TITLE
LwM2M SenML CBOR range check fix

### PR DESCRIPTION
**Pull request by VeijoPesonen**
* Updates sdk-zephyr
* Enables SenML CBOR by default.
* Enables LwM2M SEND command

### CODE REVIEW

**Review by jheiskan81 — APPROVED** [review]

**Comment by VeijoPesonen** [comment]
@jakub-uC, @nvlsianpu  Any idea what is causing the twister issue?

**Review by SeppoTakalo — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -265,6 +265,7 @@ nRF9160 samples
 
       * Reworked the retry logic so that the sample can fall back to bootstrap mode and need not always restart the LTE connection.
       * Replaced the deprecated GPS-driver with the new GNSS interface.
+      * LwM2M v1.1 uses SenML CBOR by default as content format
```
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-changelog.rst:268]:
```suggestion
      * LwM2M v1.1 uses SenML CBOR by default as content format.
```

Comment (VeijoPesonen) [reply doc/nrf/releases/release-notes-changelog.rst]: Fixed.

**Review by umapraseeda — APPROVED** [review]

**Review by SeppoTakalo — APPROVED** [review]

**Comment by SeppoTakalo** [comment]
https://github.com/nrfconnect/sdk-zephyr/pull/800 merged, so please change the west.yml to point to hash

**Review by SeppoTakalo — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Pull request merged**